src/vma/sock/sockinfo_tcp.cpp

```diff
@@ -576,6 +576,25 @@ unsigned sockinfo_tcp::tx_wait(int & err, bool is_blocking)
 	return sz;
 }
 
+unsigned sockinfo_tcp::rx_poll_once_non_blocking(int & err)
+{
```

Comment:
When using unsiged - it's for a non negative count (e.g. number of sent tx bytes)
When using int - it's for error cases (negative numbers) however in this case the errors are assigned to parameter err.
Since this method does not count anything, nor does return errors - I have changed the return value to void